lib/ansible/modules/network/aci/aci_access_port_to_interface_policy_leaf_profile.py

```diff
@@ -235,6 +241,7 @@ def main():
         'from': dict(type='str', aliases=['fromPort', 'from_port_range']),
         'to': dict(type='str', aliases=['toPort', 'to_port_range']),
         'policy_group': dict(type='str', aliases=['policy_group_name']),
+        'policy_group_type': dict(type='str', default='switch_port', choices=['port_channel', 'switch_port', 'vpc']),
```

Comment:
This was defined like this for **aci_static_binding_to_epg**:
```python
        interface_type=dict(type='str', default='switch_port', choices=['fex', 'port_channel', 'switch_port', 'vpc']),
```
@jmcgill298 @brunocalogero That parameter does have a default hardcoded there and since the parameter is required, I think we opted to add a default in this case anyway. We need a default for the lookup.

Reply:
@jmcgill298 Thanks for suggestion. I added a default for this parameter because it is a new parameter.  I did not want to break any existing playbooks that do not have this new parameter defined.

@dagwieers I'll make the variable name change. Thanks.